Dockerfile

```diff
@@ -20,20 +20,34 @@ RUN apt-get update
 RUN apt-get dist-upgrade -y
 
 # Installing packages
-RUN apt-get -y install software-properties-common
-RUN apt-get -y install bzip2 unzip openssh-client git curl zip wget
-RUN apt-get -y install lib32stdc++6 lib32z1 lib32ncurses5 lib32bz2-1.0 --no-install-recommends
-RUN apt-get -y install libxslt-dev libxml2-dev
-RUN apt-get -y install build-essential
+RUN apt-get -y install \
+  build-essential \
+  bzip2 \
+  curl \
+  git \
+  lib32stdc++6 \
+  lib32z1 \
+  lib32ncurses5 \
+  lib32bz2-1.0 \
+  libxslt-dev \
+  libxml2-dev \
+  openssh-client \
+  software-properties-common \
+  unzip \
+  wget \
+  zip \
+  --no-install-recommends
 
-# Update apt
+# Install Java
 RUN apt-add-repository ppa:openjdk-r/ppa
 RUN apt-get update
-
-# Install Java
 RUN apt-get -y install openjdk-8-jdk
```

Comment:
We have 2 issues:

* this is open source'd, so I'm not sure whether it's a good idea to put Oracle's license into an docker image we shipped under Uber's umbrella.  I will defer that to @sectioneight 
* it's advised by Google to use OpenJDK  and I believe they are moving the Android to use OpenJDK to avoid licensing issue. [https://source.android.com/source/initializing.html](info)